session/session_test.go

```diff
@@ -4019,6 +4019,18 @@ func (s *testSessionSuite) TestValidateReadOnlyInStalenessTransaction(c *C) {
 	}
 }
 
+func (s *testSessionSuite) TestTxnScopeSetting(c *C) {
```

Comment:
You removed this test in the fix test, is it a mistake?

Reply:
`TestTxnScopeSetting` is duplicated with `TestSetTxnScope`